src/js/view/controls/components/settings/menu.js

```diff
@@ -62,9 +62,9 @@ export function SettingsMenu(onVisibility, onSubmenuAdded, onMenuEmpty) {
             }
 
         },
-        close(isKeyEvent = false) {
+        close(event = {}) {
```

Comment:
You don't need a default value for `event` here. The only time a property on the event object is checked is in `visibilityChangeHandler `.